test/selenium/README.md

```diff
@@ -0,0 +1,44 @@
+# Testing Habitica with Selenium
+This folder adds selenium testing to Habitica.
+
+## Setup
+To set up the project, you will need to install Selenium Webdriver, specifically Chromedriver. Place `chromedriver.exe` in the project folder.<br>
+
+In the project folder, run the command `npm install` to install all the dependencies.<br>
+
+For most consistent results, minimize inventory and other things added to the game/account.
+
+## To run the tests
+Haibitca must be running locally first (See Habitica repo)\
+Run the command `npm test`
+
+## Manual setups
```

Comment:
This seems rather onerous for what's meant to be an automated test suite--is there a way we can create the user with the necessary data within the test database as part of running the selenium script?